.github/workflows/release.yml

```
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+
+      - name: Setup Node.js 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: npm
+
+      - name: Install Dependencies
+        run: npm i
+
+      - name: Create Release Pull Request or Publish to npm
+        uses: changesets/action@v1
+        with:
+          # No `publish` here because we're still publishing manually
+          commit: 'chore: release stylelint'
+          title: 'chore: release stylelint'
```

[suggest] Let's simplify the change and keep the commit/PR title consistent with old commits. We can change them when we will automate publishing.
It's ideal for including a version number like `Prepare 14.10.01`, but `Prepare release` seems not bad to me.

```suggestion
      - name: Create release pull request
        uses: changesets/action@v1
        with:
          commit: 'Prepare release'
          title: 'Prepare release'
```

---

I want `Prepare 14.10.01` (with version) too, but haven't find a good solution.